playground/utils/exportFiles.tsx

```diff
@@ -68,17 +51,108 @@ const indexHTML = (tabs: Tab[]) => dedent`
 </html>
 `;
 
+/**
+ * This function extracts the name and version of an imported
+ * package, returning an array [name, version], in case the
+ * import si written as name@version, or otherwhise as [name, '']
+ */
+ function extractNameAndVersionFromImportPath(importPath: string): string[] {
+    const withoutCDN = importPath.startsWith(CDN_URL)
+      ? importPath.split(`${CDN_URL}/`)[1]
+      : importPath
+    const withoutCDNSplitted = withoutCDN.split('/')
+
+    const rawName = withoutCDN.startsWith('@')
+      ? `${withoutCDNSplitted[0]}/${withoutCDNSplitted[1]}`
+      : withoutCDNSplitted[0]
+    const rawNameSplitted = rawName.split('@')
+
+    const name = rawName.startsWith('@')
+      ? `@${rawNameSplitted[1]}`
+      : rawNameSplitted[0]
+
+    const version = name.startsWith('@')
+      ? (rawNameSplitted[2] ?? '')
+      : (rawNameSplitted[1] ?? '')
+
+    return [name, version]
+  }
+
+/**
+ * This function sends a request to skypack in order to get
+ * the latest version of a pkg, which will be included in
+ * the initial comment in skypack's response
+ */
+  function getLatestVersion(pkgName: string): Promise<string[]> {
+    return fetch(`${CDN_URL}/${pkgName}`)
+      .then(res => res.text())
+      .then(data => {
+        const pkgFound = '/*\n * Skypack CDN - ' + pkgName + '@'
+        const withPkgError = '/*\n * [Package Error] "' + pkgName + '@v'
+
+        if (data.startsWith(pkgFound)) return [pkgName, data.split(pkgFound)[1].split('\n')[0]]
+        // sometmes skypack cannot compile certains packages, but will still give us the vesion
+        if (data.startsWith(withPkgError)) return [pkgName, data.split(withPkgError)[1].split('"')[0]]
```

Comment:
The [Package Error] scenario, which happens when Skypack cannot compile a package to the `esm` format if I am not wrong (e.g try with `@babel/standalone`), can slow down this process, as, when Skypack cannot build a package, the response comes considerably slower (like 1 sec).

As an alternative we could send a request to **UNPKG**, which redirects immediatly the url to the last version, making it possible to extract the version from the new url string.

However, I preferd stick to Skypack for two reasons :
1.  You can never know if Skypack and **UNPKG** ave the same reference for a package's last version. It's better if we have a single source of truth.
2. The user will see while coding the playground that the package cannot be imported, so they probably won't download it yet.